avaframe/runOperational.py

```diff
@@ -0,0 +1,92 @@
+"""
+    Run script for running the operational workflow
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+import time
+import logging
+
+# Local imports
+from avaframe.com1DFA import com1DFA
+from avaframe.log2Report import generateReport as gR
+from avaframe.out3SimpPlot import outPlotAllPeak as oP
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+from avaframe.in3Utils import initializeProject as initProj
+
+# Time the whole shebang
+startTime = time.time()
+
+# log file name; leave empty to use default runLog.log
+logName = 'runOperational'
+
+# Load avalanche directory from general configuration file
+# v0.1 -> one avalanche is fine,
+# v0.2 -> if given dir contains more than one avalancheDir -> run loop
+cfgMain = cfgUtils.getGeneralConfig()
+avalancheDir = cfgMain['MAIN']['avalancheDir']
+
+# Start logging
+log = logUtils.initiateLogger(avalancheDir, logName)
+log.info('MAIN SCRIPT')
+log.info('Current avalanche: %s', avalancheDir)
+
+log.setLevel(logging.DEBUG)
+
+# ----------------
+# Load input parameters from configuration files
+
+# ----------------
+# Clean input directory(ies) of old work and output files
+initProj.cleanSingleAvaDir(avalancheDir)
+
+# ----------------
+# Run dense flow
+# cfg = cfgUtils.getModuleConfig(com1DFA)
+# reportDictList = com1DFA.runCom1DFA(cfg, avalancheDir)
```

Comment:
This still create the old report. Is it supposed to?

Reply:
What do you mean by old report?

Reply:
the release1HX_null_dfa_0.155.html for example

Reply:
ah, @awirb correct me if I'm wrong, this comes from the generate export function of com1DFA itself, right? We can remove this, but that's not urgent... (i.e. version 0.2)

Reply:
Correct, these reports are created by the generate_report function in the run cint files of com1DFA. This function also exports the peak files.. and at the moment include more parameters than the .md report